### PR TITLE
MNT: Add noarch: python

### LINKED ISSUE #7: @conda-forge-admin, please add noarch: python
@conda-forge-admin, please add noarch: python

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've made the recipe `noarch: python` as instructed in #7.

Here's a checklist to do before merging.
- [x] Bump the build number if needed.

Fixes #7

### CODE REVIEW

**Comment by williamjamir** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,17 +10,18 @@ source:
   sha256: 877d4e080f6ded85d566127c7256de8afae6ab4fdc7ef301f7cc906a8177e3e8
 
 build:
-  skip: True  # [py<35]
-  number: 1
+  noarch: python
+  skip: True
```
Comment (CJ-Wright) [inline recipe/meta.yaml]: you need to remove this

Comment (williamjamir) [reply recipe/meta.yaml]: Commit 823735a fix remove the skip statement ;)

**Comment by CJ-Wright** [comment]
you needed to rerender after you removed the skip statement.

**Comment by williamjamir** [comment]
> you needed to rerender after you removed the skip statement.

Sorry for not letting any message here, the builds on the Azure Pipelines were not showing (even on master). I didn't know if it was a problem due to the origin of this branch (conda-forge-liner), so I decided to run the re-render in a new PR originated from this repo

https://github.com/conda-forge/sphinx-click-feedstock/pull/9

**Pull request merged**